src/js/themes/base.js

```diff
@@ -668,6 +668,17 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       // body: {
       //   extend: undefined,
       // },
+      // pinned: {
+      //   body: {
+      //      extend: undefined,
+      //   },
+      //   header: {
+      //      extend: undefined,
+      //   },
+      //   footer: {
+      //      extend: undefined,
+      //   },
+      // },
```

Comment:
Do these need background properties?

```suggestion
      // pinned: {
      //   body: {
      //      background: undefined,
      //      extend: undefined,
      //   },
      //   header: {
      //      background: undefined,
      //      extend: undefined,
      //   },
      //   footer: {
      //      background: undefined,
      //      extend: undefined,
      //   },
      // },
```